BrightnessKeys/BrightnessKeys.cpp

```diff
@@ -94,6 +94,11 @@ void BrightnessKeys::getBrightnessPanel() {
         //
         _panel = getAcpiDevice(getDeviceByAddress(info->videoBuiltin, kIOACPILCDPanel, kIOACPIDisplayTypeMask));
         
+        //
+        // On some laptops, like AMD laptops, the panel can be of legacy type.
+        //
+        _panel = getAcpiDevice(getDeviceByAddress(info->videoBuiltin, kIOACPILegacyPanel));
```

Comment:
This breaks compatibility with other devices.

Reply:
Really? Why?

Reply:
Oh, I see

Reply:
Is that what you meant?